homeassistant/components/nuki/strings.json

```diff
@@ -31,5 +31,29 @@
       "title": "Nuki webhook URL uses HTTPS (SSL)",
       "description": "The Nuki bridge can not push events to an HTTPS address (SSL), please configure a (local) HTTP address under \"Home Assistant URL\" in the [network settings]({network_link}). The current (local) address is: `{base_url}`, a valid address could, for example, be `http://192.168.1.10:8123` where `192.168.1.10` is the IP of the Home Assistant device"
     }
+  },
+  "entity": {
+    "binary_sensor": {
+      "door_sensor": {
+        "name": "[%key:component::binary_sensor::entity_component::door::name%]"
```

Comment:
Maybe we could find a better suffix than `Door` here since this would render the sensor (for example) as `Front Door Door`.
I just had a look at different integrations, they seem to either dismiss the name on the door sensor or use `state`, but switchbot also has this configured as `[%key:component::binary_sensor::entity_component::door::name%]`

Reply:
Maybe leave out the entity name for this sensor? (set it to `None`)?

Reply:
Done